src/Core/src/Handlers/FlyoutView/FlyoutViewHandler.Android.cs

```diff
@@ -255,8 +255,6 @@ void LayoutAsFlyout()
 				DrawerLayout.AddView(flyoutView, layoutParameters);
 			}
 
-			DrawerLayout.CloseDrawer(flyoutView);
```

Comment:
Does this mean that the flyout will now never close whenever we select another item? I don't think that is what we want.

Reply:
@jfversluis, Currently, the flyout is not closed by default when selecting items in the flyout view across all platforms.

In the reported scenario, the issue arises when the user [updates](https://github.com/Tamilarasan-Paranthaman/maui/blob/83c1bff3a319706ee0996414a892aa022378c9d8/src/Controls/tests/TestCases.HostApp/Issues/XFIssue/Issue973.cs#L66) the **FlyoutPage.Details** while selecting an item in the flyout. This triggers the flyout to close by default. However, it does not close on other platforms, leading to inconsistency across the platforms. Could you please share your insight on this scenario?